### DISCUSSION TITLE
Ble Pairing with Shiny

### DISCUSSION THREAD

**Discussion by BillyMartin1964** [shinyorg/shiny#1539] [Q&A]
I haven't been able to find much info about Pairing with shiny Ble. Does anyone have any links or sample code on this, please? I am using .Net Maui Android and iOS. Thanks.

**Comment by aritchie** [comment]
There is no pairing on iOS.  If you look in the source code, you'll see exactly what you're looking for with regards to Android: https://github.com/shinyorg/shiny/blob/62a033f12902ed2c6bc5145ab60ecdc0e7d59663/src/Shiny.BluetoothLE/Feature_Pairing.cs#L41

**Reply by BillyMartin1964** [reply]
Thanks. Somehow overlooked it in my search.
